doc/api/net.md

```diff
@@ -725,7 +725,7 @@ added:
 * `ip` {string} The IP which the socket attempted to connect to.
 * `port` {number} The port which the socket attempted to connect to.
 * `family` {number} The family of the IP. It can be `6` for IPv6 or `4` for IPv4.
-  \* `error` {Error} The error associated with the failure.
+  \* `error` {Error} The error associated with the failure.
```

Comment:
Something's off here, is `error` the fourth parameter, or is it just out of place?